certificate-policy.md

```diff
@@ -168,42 +169,44 @@ The key words "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMME
 
 
 # 2. PUBLICATION AND REPOSITORY RESPONSIBILITIES
-The FPKIPA will review and update this Certificate Policy at least every 365 days to ensure compliance with CAB Forum Baseline Requirements. After review and approval, the CP document version number and a dated changelog entry shall be added even if no changes were deemed necessary.
-
-The review and update shall include any changes needed to address:
-
-- U.S. Federal Government mission needs and changes to procedures to support the missions
-- CAB Forum Baseline Requirements updates
-
-If changes to CAB Forum Baseline Requirements are made and have applicable requirements which require compliance earlier than 365 days from the last update, the FPKIPA will update the policy to meet those compliance timeframes and ensure all CA's and associated CA Certification Practice Statements are updated.  
-
-Each CA shall develop, implement, enforce, and update at least every 365 days a Certification Practice Statement (CPS) that describes in detail how the CA implements the requirements of this CP.
-
-An annual self-assessment shall be conducted by any CA operating under this Certificate Policy and the accompanying CA CPS to show compliance with the latest version of this certificate policy and the CAB Forum Baseline Requirements. The CA shall indicate the self-assessment by incrementing the CPS version number and adding a dated changelog entry to the change record.
 
 ## 2.1 Repositories
-The CA shall make revocation information for Subordinate Certificates and Subscriber Certificates available in accordance with this Policy.
+Each CA shall disclose the following information through a publicly accessible Repository: 
+- CA Certificates
+- Certificate Revocation Lists (CRLs)
+- Online Certificate Status Protocol responses
+- CPS documents
+- Subscriber Agreements
+- Audit Reports
 
-The CA shall post all CA certificates and CRLs issued by the CA in a repository that is publicly accessible through all Uniform Resource Identifier (URI) references asserted in valid certificates issued by that CA. The CA shall document the Uniform Resource Identifiers for CA certificates and CRLs in the CPS.
+Each CAs shall ensure that its root Certificate and the revocation data for issued Certificates are available through a Repository 24 hours a day, 7 days a week with a minimum of 99% availability overall per year with a scheduled down time that does not exceed 0.5% annually. 
```

Comment:
Then what we're really saying is `99.5%` availability, not `99%`.

Reply:
This was for two different submitted comments.   It's in a handful of other CPs (non-govt) as well as other govt CPs.    

It's the clause "scheduled down time" that is being used to distinguish and is from a historical approach.  For example, a _scheduled_ patching cycle versus _unscheduled_ downtime due to natural disasters or other factors.   

I agree with you though; it's confusing in **today's** technology environment.  I'll update and we'll see what everyone thinks.

Reply:
Ah, you're right, I didn't notice the distinction, and I can see how you would make it. Going to straight 99.5% works, or maybe just a rephrasing to make it more obvious to dunces like me who read too fast. :)